docs/create_recipe_template.mdx

```diff
@@ -0,0 +1,86 @@
+---
+title: "Creating with SparseML Recipe Template"
+metaTitle: "Creating Sparsification Recipes with SparseML"
+metaDescription: "Creating Sparsification Recipes with SparseML"
+index: 1000 
+---
+
+# Creating Sparsification Recipes with SparseML Recipe Template
+
+This page explains how to create recipes using the SparseML Recipe Template local API. 
+
+The SparseML Recipe Template API is a simple API that outtakes in an optional path to a dense Pytorch model as input and returns a [Sparsification recipe](https://docs.neuralmagic.com/user-guide/recipes) for that given model in the form of a `recipe.md` file. If no model is supplied, SparseML Recipe Template will generate a default Sparsification Recipe for a generic model. The Sparsification recipe that is outputted by a call to the SopaseML Recipe Template API can then be applied to a model in [Pytorch, Keras, or TensorFlow V1 training pipelines](https://docs.neuralmagic.com/user-guide/recipes/enabling) to train asparse model, all with just a few lines of code. 
+
```

Comment:
```suggestion
SparseML Recipe Template is a simple API to generate sparsification recipes on the fly. In the most basic use case it takes in information about the pruning and quantization techniques and produces a relevant recipe.
Optionally path to a `torch.jit` loadable model can also be specified in which case the recipe produced is specific to the architecture of the specified model.  The  sparsification recipes are produced in `markdown` or `yaml` format which can be directly used across all SparseML utilities. Learn more about sparsification recipes [here](https://docs.neuralmagic.com/user-guide/recipes)
```

The reason being we can also specify a sparse model in recipe template, and the code as of now does not support application to Tensorflow, or Keras models